$NetBSD: patch-build__tools_make__xml__documentation,v 1.1 2023/12/15 14:18:16 gdt Exp $

Fix non-POSIX \b in sed to POSIX character class.

Not yet reported upstream.
(A bracketed [[:space:]]: GNU sed rejects a bare [:space:].)

--- build_tools/make_xml_documentation.orig
+++ build_tools/make_xml_documentation
@@ -220,7 +220,7 @@
 					${XMLSTARLET} val -e -d "${source_tree}/doc/appdocsxml.dtd" "${i}" || { echo "" ; exit 1 ; }
 			fi
 		fi
-		${SED} -r "/^\s*(<[?]xml|<.DOCTYPE|<.?docs)/d" "${i}" >> "${output_file}"
+		${SED} -r "/^[[:space:]]*(<[?]xml|<.DOCTYPE|<.?docs)/d" "${i}" >> "${output_file}"
 	done
 done
 echo "</docs>" >> "${output_file}"
